code/backtracking/src/number_of_ways_in_maze/number_of_ways_in_maze.cpp

```diff
@@ -8,7 +8,7 @@ bool solveMaze(char maze[][10],int sol[][10],int i,int j,int m,int n,int &ways){
     if(i==m && j==n){
         sol[m][n] = 1;
         ways++;
-        ///Print the soln
+        ///Print the solution
```

Comment:
One thing I would like to change is the usage of triple slash comments. Could you remove 1 slash from each of them? Also consider putting a space between the comment and the text.

Reply:
Sorry I didn't notice that. Sure will do it right away.